.goreleaser.yaml

```diff
@@ -0,0 +1,3 @@
+release:
+  header: |
+    Check the [v{{.Version}} contrib changelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tags/v{{.Version}}) and the [v{{.Version}} core changelog](https://github.com/open-telemetry/opentelemetry-collector/releases/tags/v{{.Version}}) for changelogs on specific components.
```

Comment:
Curious, could this be added to each distro's goreleaser file with its specific link?

Reply:
It can't, not like this anyways. I tried it, the "header" property throws an error or gets ignored on release.
This default .goreleaser only functions when it's at the root of the project folder.

Reply:
Here's a test repository I created to show how it works, I made a release on there to be sure the formatting was correct. 
Here's a link to it : https://github.com/Enzujp/release-sample

Reply:
@Enzujp have you tested how this file at the root will interact with our release process which specifies a specific `workdir` that isn't root?

Reply:
Yes I did, and the goreleaser docs emphasizes the same thing, regardless of the `workdir` the .goreleaser file at the root gets triggered on release and appends the custom formatted header which we've defined in the code.

Reply:
@Enzujp can you update your test repo to be structured like ours and show that test?

Reply:
@TylerHelmuth Sure, I used the folder structure and included the .goreleaser files in the `distributions directory`, took out the authentication bits since it's a demo and just ran it as a skeleton to test if it would all work as it should, the failed tests are as a result of the absent bits requiring auth, but it shows that everything runs perfectly fine with the default .goreleaser file at the root of the project, and the header is functional still as you can see from the release. 
Here's the link : https://github.com/Enzujp/sample-releases

Reply:
Sweet, thank you!

Reply:
My pleasure! Could you recommend another issue I could work on please ? @TylerHelmuth